.github/workflows/ci_image_build.yml

```diff
@@ -16,6 +16,8 @@ jobs:
 
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
```

Comment:
Why not just list arm64? That's what i did on https://github.com/timescale/timescaledb-toolkit/blob/eg/aarch64-image/.github/workflows/ci_image_build.yml